src/libcore/num/f32.rs

```diff
@@ -219,9 +219,15 @@ impl Float for f32 {
     /// Computes the absolute value of `self`. Returns `Float::nan()` if the
     /// number is `Float::nan()`.
     #[inline]
+    #[cfg(stage0)]
     fn abs(self) -> f32 {
         unsafe { intrinsics::fabsf32(self) }
     }
+    #[inline]
+    #[cfg(not(stage0))]
+    fn abs(self) -> f32 {
+        unsafe { intrinsics::fabs(self) }
+    }
```

Comment:
I think that these may want to be inverted because currently this means that the `not(stage0)` functions all have no documentation.